frontend/src/routes/Home/index.tsx

```diff
@@ -24,8 +25,15 @@ export default function Home() {
     dispatch(getUpdateMockString())
   }
 
+  const Reddropdownbar = styled(DropDownBar)`
```

Comment:
Consider abstracting this into our `dropdown` "reusable" components: since throughout the app we're likely red-ing it!

Reply:
Removed in commit 5d3a88d93cd2292ee86164198444021046833cc7, Select component will be used instead of dropdown (by @limwenfeng in laundryCard branch)